features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher-new/.eslintignore

```diff
@@ -6,4 +6,5 @@
 /source/src/app/components/Apis/Details/LifeCycle/**
 /source/src/app/components/Apis/Details/Documents/**
 /source/src/app/components/Apis/Details/Endpoints/**
+/source/src/app/components/Apis/Details/NewOverview/**
```

Comment:
How about renaming the old Overview as OldOverview and keep the new one as "Overview" itself. Then we can delete the OldOverview files easily later without the need to change the code.